html5/index.html

```diff
@@ -678,7 +679,14 @@ <h2>Xpra Bug Report</h2>
 				client.remote_logging = remote_logging;
 				client.sharing = sharing;
 				client.insecure = insecure;
-				client.clipboard_enabled = clipboard;
+				// TODO: Re-enable clipboard support for Safari when it implements the async clipboard API.
+				//       See https://medium.com/nerd-for-tech/navigator-clipboard-api-b96be187df5d
+				if (Utilities.isSafari() && clipboard) {
+					clog("disabling clipboard support on Safari");
+					client.clipboard_enabled = false;
+				} else {
+					client.clipboard_enabled = clipboard;
+				}
```

Comment:
Why disable the clipboard completely?
Does the clipboard not work at all on Safari? Not even in fallback mode?

Normally, if the browser doesn't support the async clipboard API, we switch back to the old API - which isn't great, but kinda works - better than nothing?